app/views/budgets/investments/_ballot.html.erb

```diff
@@ -59,7 +59,7 @@
                 verify_account: verify_account, signin: signin,
                 signup: signup, my_heading: my_heading,
                 change_ballot: change_ballot,
-                heading_link: heading_link(@assigned_heading, @budget)).html_safe %>
+                heading_link: sanitize heading_link(@assigned_heading, @budget)).html_safe %>
```

Comment:
@cartothemax Thanks for having a look at this! Actually I'm preparing a pull request which will remove all usages of `.html_safe`. Sorry I didn't tell you :pray:; I thought you were only going to work on the redirect warnings.

So, if it isn't asking too much, could you remove the commit dealing with XSS issues (with `git rebase`)? That way our pull requests won't have any conflicts :thinking:.

I'll add a link to the pull request when I open it, so you can review it if you'd like to :wink:.

Reply:
FYI, I've opened pull request #3747, although it still needs some work.

Reply:
No problem! I got a little carried away chasing down all of the Brakeman errors rather then just sticking to the redirect warnings like the issue stated 🤣